### ISSUE TITLE
@loaders.gl/geotiff Documentation

### ISSUE THREAD

**Issue by im28** [visgl/loaders.gl#2239]
Hi, I found that one of the packages I would like to use is missing documentation.
@loaders.gl/geotiff Package Exists yet there are no documentation or usage details for it on the website

**Comment by ibgreen** [comment]
Correct. The geotiff module is still experimental and unfortunately the effort to complete the work lost steam (mainly due to conflicts on my time). IIRC, one of the bigger issues is that the API is not fitting well with the loaders.gl model and there would need to be some work designing adapter code.

**Comment by im28** [comment]
Thanks for the explanation, I just thought it would be really cool if a Cloud Optimized Geotiff could be directly used in deck.gl via a loader. I will try to read up more since I'm not that familiar with the loaders.gl model.

**Comment by ibgreen** [comment]
Agreed, that would be extremely cool! 

As you plan your tactics, keep in mind that GeoTIFF is a complicated format. Viewing a GeoTIFF can require mapping a collections of bands to RFB colors, and large geotiffs are optimized for using range requests to read out hierarchical "sub chunks". Also the GeoTIFF can be in a highly non-mercator projection which needs delicate reprojection.
